error-prone-contrib/src/main/java/tech/picnic/errorprone/refasterrules/OptionalRules.java

```diff
@@ -282,6 +282,8 @@ T after(Optional<T> optional, T value) {
   // XXX: The rewritten `filter`/`map` expression may be more performant than its replacement. See
   // https://github.com/palantir/gradle-baseline/pull/2946. (There are plans to pair Refaster rules
   // with JMH benchmarks; this would be a great use case.)
+  // XXX: Perhaps `stream.mapMulti(Optional::ifPresent)` is what we should use. See
+  // https://github.com/palantir/gradle-baseline/pull/2996.
   static final class StreamFlatMapOptional<T> {
```

Comment:
I didn't mean to add this to this PR, but since it's here: let's just include. Updated the suggested commit message.

CC/FYI @werli.